demos/mlsys_2020/demo_handler.py

```diff
@@ -0,0 +1,137 @@
+from demo_source import *
+
+myDemo = Demo()
```

Comment:
I'm wondering if we can put all of this code behind a function, so that we limit the amount of top-level definitions?

Some reasons for this are:
1. It reduces the amount of top-level definitions and thus makes it less likely to have collisions.
2. Being able to import this module can avoid the frowned upon `import module from *` form. You can instead say
```
import demo
demo.run()
```

Reply:
Good point. I agree with you.